src/site/constants/environments-configs.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * All possible values for the environment name, also known as the build-type.
+ * @module site/constants/environments
+ */
+const ENVIRONMENTS = require('./environments');
+
+module.exports = {
+  [ENVIRONMENTS.VAGOVPROD]: {
+    BUILDTYPE: ENVIRONMENTS.VAGOVPROD,
+    BASE_URL: 'https://www.va.gov',
+    API_URL: 'https://api.va.gov',
+  },
+
+  [ENVIRONMENTS.VAGOVSTAGING]: {
+    BUILDTYPE: ENVIRONMENTS.VAGOVSTAGING,
+    BASE_URL: 'https://staging.va.gov',
+    API_URL: 'https://staging-api.va.gov',
+  },
+
+  [ENVIRONMENTS.VAGOVDEV]: {
+    BUILDTYPE: ENVIRONMENTS.VAGOVDEV,
+    BASE_URL: 'https://dev.va.gov',
+    API_URL: 'https://dev-api.va.gov',
+  },
+
+  [ENVIRONMENTS.LOCALHOST]: {
+    BUILDTYPE: ENVIRONMENTS.LOCALHOST,
+    BASE_URL: 'http://localhost:3001',
```

Comment:
This is duplicate code from the environments helper. The helpers has several references to jsdom that are not available globally in our build process.

I'm open to ideas on how we want to do this. [This](https://github.com/department-of-veterans-affairs/va.gov-team/issues/1153) was suggested and I'd be willing to take this on within a couple of sprints if we want to go the direction of using a dotenv file

Reply:
Yeah, what @cvalarida said. It's gonna be really confusing to have these in more than one place.